storage/innobase/xtrabackup/src/changed_page_tracking.cc

```diff
@@ -170,21 +188,14 @@ bool get_changed_pages(lsn_t start_lsn, lsn_t end_lsn, uint64_t *backupid,
     return (false);
   }
 
-  std::string udf_get_start_lsn = Backup_comp_constants::udf_get_start_lsn;
-
   /* validate tracking lsn */
-  char *start_lsn_str = read_mysql_one_value(
-      connection, ("SELECT " + udf_get_start_lsn + "()").c_str());
-  lsn_t backup_start_lsn = strtoll(start_lsn_str, nullptr, 10);
-  free(start_lsn_str);
+  lsn_t backup_start_lsn = get_pagetracking_start_lsn(connection);
```

Comment:
where do we do the reset()? mysqlbackup_page_track_set()?

Reply:
yes. It is Inside the **start** method define in src/changed_page_tracking.cc